app/src/components/layout/Toolbar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <v-app-bar flat color="#00254f">
+    <v-app-bar flat height="100vh" color="#00254f">
```

Comment:
100vh would be the total visible vertical space @see https://www.w3schools.com/cssref/css_units.asp
I think the v-app-bar interprets this as a px value. Removing the vh from the 100 does not change the behaviour (even 100bananas has the same result).
Please remove the unit or use the correct one.

Reply:
You're absolutely right 😄 I checked out the older version which contained the vh unit. Just changed it to px